router/xlink_transport/listener.go

```diff
@@ -224,7 +224,7 @@ func (self *listener) bindNonSplitChannel(binding channel.Binding, routerId, rou
 
 	if existingLink, applied := self.xlinkRegistery.LinkAccepted(xli); applied {
 		log.Info("link registered")
-	} else {
+	} else if existingLink != nil {
```

Comment:
does it make sense to keep an actual else indicating neither match for "unexpected situations'?

Reply:
The other condition (not applied, no existing link) is logged in LinkAccepted